bertopic/representation/_langchain.py

```diff
@@ -1,23 +1,26 @@
 import pandas as pd
-from tqdm import tqdm
-from scipy.sparse import csr_matrix
-from typing import Mapping, List, Tuple, Union, Callable
 from langchain.docstore.document import Document
+from scipy.sparse import csr_matrix
+from typing import Callable, Dict, Mapping, List, Tuple, Union
+
 from bertopic.representation._base import BaseRepresentation
 from bertopic.representation._utils import truncate_document
 
-
 DEFAULT_PROMPT = "What are these documents about? Please give a single label."
 
 
 class LangChain(BaseRepresentation):
     """ Using chains in langchain to generate topic labels.
 
-    Currently, only chains from question answering is implemented. See:
-    https://langchain.readthedocs.io/en/latest/modules/chains/combine_docs_examples/question_answering.html
+    The classic example uses `langchain.chains.question_answering.load_qa_chain`.
+    This returns a chain that takes a list of documents and a question as input.
+
+    You can also use Runnables such as those composed using the LangChain Expression Language.
 
     Arguments:
-        chain: A langchain chain that has two input parameters, `input_documents` and `query`.
+        chain: A langchain chain or Runnable with a `batch` method.
+            Input keys must be `input_documents` and `question`.
+            Output key must be `output_text`.
```

Comment:
Could you update the docstrings such that it matches with the other arguments?

Reply:
I think 8d96b04f126d1dcbd33b7afeedcad8a7f0ccb73f improves the readability, but please feel free to add changes if it's not quite what you wanted. :) Thank you.